README.md

```diff
@@ -7,6 +7,7 @@
 [![Maven Central](https://img.shields.io/maven-central/v/com.alibaba.csp/sentinel-core.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:com.alibaba.csp%20AND%20a:sentinel-core)
 [![License](https://img.shields.io/badge/license-Apache%202-4EB1BA.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
 [![Gitter](https://badges.gitter.im/alibaba/Sentinel.svg)](https://gitter.im/alibaba/Sentinel)
+[![Leaderboard](https://img.shields.io/badge/Sentinel-Check%20Your%20Contribution-orange)](https://opensource.alibaba.com/contribution_leaderboard/details?projectValue=sentinel)
```

Comment:
`Alibaba%20Sentinel-%20Contribution%20Leaderboard-orange` maybe is better than `Sentinel-Check%20Your%20Contribution-orange` 😃

Reply:
Maybe, but we think `Check your contribution` may attract more attention from developers since `leaderboard` is not quite a familiar word to Chinese developer, and for the repos with Chinese documentation, we use `点击查看你的贡献`.